src/GraphQL/Builders/ConnectionBuilder_Typed.cs

```diff
@@ -0,0 +1,291 @@
+        public static ConnectionBuilder<TSourceType, TReturnType> Create<TNodeType, TEdgeType>(string name = "default")
+            where TNodeType : IGraphType
+            where TEdgeType : EdgeType<TNodeType>
+            => Create<TNodeType, TEdgeType, ConnectionType<TNodeType, TEdgeType>>(name);
+
+        /// <summary>
+        /// Returns a builder for new connection field for the specified node, edge and connection type.
+        /// </summary>
+        /// <typeparam name="TNodeType">The graph type of the connection's node.</typeparam>
+        /// <typeparam name="TEdgeType">The graph type of the connection's edge. Must derive from <see cref="EdgeType{TNodeType}">EdgeType</see>&lt;<typeparamref name="TNodeType"/>&gt;.</typeparam>
+        /// <typeparam name="TConnectionType">The graph type of the connection. Must derive from <see cref="ConnectionType{TNodeType, TEdgeType}">ConnectionType</see>&lt;<typeparamref name="TNodeType"/>, <typeparamref name="TEdgeType"/>&gt;.</typeparam>
+        public static ConnectionBuilder<TSourceType, TReturnType> Create<TNodeType, TEdgeType, TConnectionType>(string name = "default")
+            where TNodeType : IGraphType
+            where TEdgeType : EdgeType<TNodeType>
+            where TConnectionType : ConnectionType<TNodeType, TEdgeType>
+        {
+            var fieldType = new FieldType
+            {
+                Name = name,
+                Type = typeof(TConnectionType),
+                Arguments = new QueryArguments(),
+            };
+            fieldType.Arguments.Add(new QueryArgument(typeof(StringGraphType))
+            {
+                Name = "after",
+                Description = "Only look at connected edges with cursors greater than the value of `after`.",
+            });
+            fieldType.Arguments.Add(new QueryArgument(typeof(IntGraphType))
+            {
+                Name = "first",
+                Description = "Specifies the maximum number of edges to return, starting after the cursor specified by `after`, or the first number of edges if `after` is not specified.",
+            });
+
+            return new ConnectionBuilder<TSourceType, TReturnType>(fieldType);
+        }
+
+        /// <summary>
+        /// Configure the connection to be bi-directional.
+        /// </summary>
+        public ConnectionBuilder<TSourceType, TReturnType> Bidirectional()
+        {
+            if (_isBidirectional)
+                return this;
+
+            Argument<StringGraphType, string>("before",
+                "Only look at connected edges with cursors smaller than the value of `before`.");
+            Argument<IntGraphType, int?>("last",
+                "Specifies the number of edges to return counting reversely from `before`, or the last entry if `before` is not specified.");
```

Comment:
This comment also looks weird.

Reply:
I agree.

Reply:
I agree but don’t have time to think about it. It’s not always easy to write text. I read it but nothing came to mind.

Maybe if I have time I’ll work on it. But I’ve not even been able to merge your prior suggestions in the last 4 days.

Reply:
"Specifies the maximum number of edges to return counting reversely from the cursor specified by `before`, or the last number of edges if `before` is not specified.");